docs/source/advanced/language-models.rst

```diff
@@ -1,45 +1,67 @@
 .. _language_models:
 
-Use Text as Node Features
-=============================
-Many real world graphs have text contents as nodes' features, e.g., the title and description of a product, and the questions and comments from users. To leverage these text contents, GraphStorm supports language models (LMs), i.e., HuggingFace BERT models, to embed text contents and use these embeddings in Graph models' training and inference.
+Use Language Models in GraphStorm
+==================================
+Many real world graphs have text contents as nodes' features, e.g., the title and description of a product, and the questions and comments from users. To leverage these text contents, GraphStorm supports multiple language models (LMs) from HuggingFace to embed text contents and use these embeddings for various Graph Machine Learning tasks.
 
-There are two modes of using LMs in GraphStorm:
+There are a set of modes to use LMs in GraphStorm.
 
-* Embed text contents with pre-trained LMs, and then use them as the input node features, without fine-tuning the LMs. Training speed in this mode is fast, and memory consumption will be lower. However, in some cases, pre-trained LMs may not fit to the graph data well, and fail to improve performance.
+#. Use Pre-trained LMs only without fine-tuning
+    In this mode, users can embed text contents with pre-trained LMs, and then use them as the input node features to train GNN models, but do not fine-tune the LMs. Model training speed in this mode is fast, and memory consumption will be lower. However, in some cases, pre-trained LMs may not fit to the graph data well, hence not improving performance.
```

Comment:
Suggest changing "pre-trained LMs may not fit to the graph data well" to "pre-trained LMs may not fit to the text data on graph"

Reply:
thanks, changed.